tests/utils/index.js

```diff
@@ -1038,10 +1038,8 @@ const saveLogs = async () => {
   }
 };
 
-const tearDownServices = async (removeOrphans) => {
-  if (removeOrphans) {
-    return dockerComposeCmd('down', '-t', '0', '--remove-orphans', '--volumes');
-  }
+const tearDownServices = async () => {
+  await dockerComposeCmd('down', '-t', '0', '--remove-orphans', '--volumes');
```

Comment:
I have added this on purpose, because I frequently inspect the existent containers to debug e2e tests. 
I would prefer not to remove it.

Reply:
:laughing: You are a step ahead of me here!  I was going to solicit your help with this [[Chesterton's Fence](https://en.wiktionary.org/wiki/Chesterton%27s_fence)](https://en.wiktionary.org/wiki/Chesterton%27s_fence) once I determined if this was actually required for the CI or not....

My main complaint with the current behavior is that it results in an additional CHT instance that is constantly running on my local machine (that does not get torn down until the *next time* I run the e2e/integration tests).  For me, this is a somewhat minor annoyance (since my laptop is well provisioned), but this could be a bigger issue for someone with less system resources.  Additionally, it just seems like a bad practice to leave the test instance running *by default*.  An external contributor to the CHT might not be super familiar with Docker and may not even realize that, by running the tests locally, they will have a CHT instance eternally running on their computer....

At the _very least_ it seems that we should print a message when the tests complete that includes the command to run for tearing down the instance.

How would you feel about removing the containers by default, but skipping this command and keeping the containers if `process.env.DEBUG` is truthy?  (Seems to align well the the other debugging functionality in the tests that get toggled with that flag....)

Reply:
That would work quite well, actually :)